pulitzer.gemspec

```diff
@@ -32,6 +32,9 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'sqlite3', '~> 1.3'
   s.add_development_dependency 'minitest', '~> 5.1'
   s.add_development_dependency 'byebug', '~> 8.0'
+  s.add_development_dependency 'pry-rails', '~> 0.3', '< 1.0'
```

Comment:
For the exact same reason: why would I spend time learning Byebug when Pry does everything just as well and I already know it? This is why we don't add new tools that do the same thing. I'd like to remove Byebug to maintain consistency across all our projects and reduce the need to learn multiple things that work the same.